<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DtoD</class>
 <widget class="QDialog" name="DtoD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="URL">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>URL : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DCcon to DISCORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Index Option</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MIN : </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MIN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>321</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MAX:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MAX">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>99</number>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDir">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Directory Option</string>
   </property>
  </widget>
  <widget class="QLabel" name="showDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Directory : None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveMM">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorornot">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WORKING .....</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Readme">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Readme</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>if you dont know use this</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>╰─&gt; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="channel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Channel ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Discord Token :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnalysisToken">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>input</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
